client/components/jetpack-logo/index.jsx

```diff
@@ -1,35 +1,58 @@
 /** @format */
+
 /**
  * External dependencies
  */
 import React from 'react';
+import PropTypes from 'prop-types';
 
-const JetpackLogo = ( { size, full } ) => {
+/**
+ * Module constants
+ */
+const logoPathSize32 = (
+	<path
+		className="jetpack-logo__icon"
```

Comment:
Can we remove this? It's unused at the moment and only seems relevant for the `full` usage.

Reply:
@sirreal We can, but I left it in for occasions where we want the icon green.